cepgen.spec

```diff
@@ -0,0 +1,35 @@
+### RPM external cepgen 1.0.1
+
+Source: https://github.com/cepgen/cepgen/archive/refs/tags/%{realversion}.tar.gz
+
+BuildRequires: cmake ninja hepmc hepmc3 lhapdf pythia6
+
+%prep
+%setup -n %{n}-%{realversion}
+
+%build
+rm -rf ../build
+mkdir ../build
+cd ../build
+
+export HEPMC_DIR=${HEPMC_ROOT}
+export HEPMC3_DIR=${HEPMC3_ROOT}
+export LHAPDF_DIR=${LHAPDF_ROOT}
```

Comment:
looks like you need `LHAPDF_PATH` ( see https://github.com/cepgen/cepgen )

Reply:
Thanks a lot! You know my own code more than myself 😄